infrastructure/main.tf

```diff
@@ -1,3 +1,54 @@
 provider "azurerm" {
   features {}
 }
+
+locals {
+  tags = merge(var.common_tags,
+    map(
+      "environment", var.env,
+      "managedBy", var.team_name,
+      "Team Contact", var.team_contact,
+    )
+  )
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = "${var.product}-${var.component}-${var.env}"
+  location = var.location
+  tags     = local.tags
+}
+
+data "azurerm_user_assigned_identity" "et-identity" {
+  name                = "${var.product}-${var.env}-mi"
+  resource_group_name = "managed-identities-${var.env}-rg"
+}
+
+data "azurerm_key_vault" "s2s_vault" {
+  name                = "s2s-${var.env}"
+  resource_group_name = "rpe-service-auth-provider-${var.env}"
+}
+
+data "azurerm_key_vault_secret" "et_sya_s2s_key" {
```

Comment:
I would prefer to call it et_sya_api_s2s_key just to keep a consistent naming

Reply:
Sounds good! Will rename!